packages/react-rest/src/services/useDeleteOne/useDeleteOne.test.ts

```diff
@@ -32,37 +32,26 @@ describe("react-rest/services/useDeleteOne", () => {
     )
 
     await waitFor(() => {
-      expect(result.current).toEqual([
-        expect.any(Function),
-        {
-          status: "success",
-          meta: undefined,
-          error: undefined,
-          isResolved: true,
-          isPending: false,
-          isRejected: false,
-          isRevalidating: false,
-          isStale: false,
-          isSuccess: true,
-        },
-      ])
+      expect(result.current).toEqual([expect.any(Function), {}])
     })
 
-    await result.current[0]("id")
+    result.current[0]("id")
```

Comment:
Good catch, I believe I removed at forgot to put it back. It should be returning a promise

Reply:
Assigning this to a value causes failures in the test, but I think I'll leave the result.current because that is the destructured return value from the renderHook() call. I don't want it to be confused with the actual return of the `useDeleteOne()` hook.